python/setup.py.in

```diff
@@ -174,6 +174,18 @@ if '${CMAKE_BUILD_TYPE}' == 'Release':
             raise Exception("patch libmkldnn.so failed, command: %s" % command)
         package_data['paddle.libs']+=['libmkldnn.so.0']
         shutil.copy('${MKLDNN_SHARED_LIB}', libs_path)
+if '${WITH_NGRAPH}' == 'ON':
+    if '${CMAKE_BUILD_TYPE}' == 'Release':
```

Comment:
This is duplicated with  Line163, could be placed in the Line 163

Reply:
addressed in PR https://github.com/PaddlePaddle/Paddle/pull/14441